integration_test/third_party_apps_data/applications/vault/metadata.yaml

```diff
@@ -4,6 +4,117 @@ long_name: Hashicorp Vault
 description: |-
   Vault is an identity-based secrets and encryption management system. This integration Collects Vault's audit logs.
```

Comment:
Since we're adding the metrics receiver here, shouldn't this summary also say something about metrics. Perhaps something like this:

... audit logs. The integration also collects token, memory, and storage metrics.

Reply:
Updated